### ISSUE TITLE
Curriculum Review: Day 4 Discrimination in software

### ISSUE THREAD

**Issue by PauliPalbi** [semaphoreP/codeastro#66]
**Content:**

About the slides:
In general I think the topics covered are very interesting. I would put less text on the slides and more images. 

To make it more interactive with the participants I would suggest to start with a game/survey or something like that to test their discrimination. It is always good to expose how discrimination is deeply inside us even if we think we question it.

**Comment by semaphoreP** [comment]
Thanks for the comments! I tried to make the slides a bit less text heavy by adding some diagrams. 

Maybe we can ask everyone to take one of the implicit bias tests (https://implicit.harvard.edu/implicit/selectatest.html), and poll for results? What do you think?

**Comment by PauliPalbi** [comment]
Slides looks good :) and at least I like a test like those and then poll the results, but it should be a short one, have you already take one of them? (I did some time ago but it was way too long). And how will we get the results from the participants?

**Comment by semaphoreP** [comment]
Yes, I took the Science-Gender IAT test and it only took me about 6 minutes to complete. I've blocked 10 minutes at the start to do this to get people thinking about implicit bias. We'll do an anonymous poll where everyone can mark what they got (one of 7 possible results). Here is the result from their study: 

![image](https://user-images.githubusercontent.com/2245740/122457468-89783300-cf63-11eb-9977-ebd5f8c22aed.png)
